formats/detailed_guide/publisher/links.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "topics",
```

Comment:
These should be required in details.json instead, as that's where topics now live.